player/screenshot.c

```diff
@@ -127,6 +127,30 @@ static void append_filename(char **s, const char *f)
     talloc_free(append);
 }
 
+static void trim_invalid_utf8(char *s, size_t len)
+{
+    for (int i = 1; i <= 3; i++) {
+        if (bstr_parse_utf8_code_length(s[len-i]) > i) {
+            s[len-i] = 0;
+            return;
+        }
+    }
+}
```

Comment:
this doesn't make much sense on its own and should be inlined into the other function